### PR TITLE
fix: eslint configuration for windows

### PR DESCRIPTION
**Pull request by sdo-1A**
## Proposed change

Fix configuration of eslint to work for windows environment (absolute file paths and quotes in nx.json)

## Related issues

- :bug: Fixes #(issue)
- :rocket: Feature #(issue)

### CODE REVIEW

**Review by matthieu-crouzet — APPROVED** [review]

File: nx.json [hunk 1]
```diff
@@ -117,11 +117,11 @@
     "lint": {
       "options": {
         "cwd": "{projectRoot}",
-        "command": "yarn eslint '**/*.{m,c,}{j,t}s' '**/*.html' '**/*.json' --no-error-on-unmatched-pattern"
+        "command": "yarn eslint \"**/*.{m,c,}{j,t}s\" \"**/*.html\" \"**/*.json\" --no-error-on-unmatched-pattern"
```
[the remark this answers is not in the document]
Comment (matthieu-crouzet) [reply nx.json:120]: `'` were not working on Windows

previously we were using [@nx/eslint:lint](https://nx.dev/nx-api/eslint/executors/lint) where we had to set `lintFilePatterns` I replaced it with that to have the same command on each package (we can still configure it for each if needed)
or we can rediscussed it but it should not be part of this PR

File: tools/github-actions/audit/packaged-action/index.js [hunk 2]
```diff
@@ -26189,9 +26189,9 @@ var __importStar = (this && this.__importStar) || function (mod) {
     return result;
 };
```
Comment (cpaulve-1A) [inline tools/github-actions/audit/packaged-action/index.js:26190]: It seems strange only generated files have been modified in this PR and not the original main.ts?

Comment (matthieu-crouzet) [reply tools/github-actions/audit/packaged-action/index.js:26190]: She modified the nx.json, so it invalidated the cache, regenerated the code and linted it

Comment (fpaul-1A) [reply tools/github-actions/audit/packaged-action/index.js:26190]: why are we even linting the generated js code ?

Comment (cpaulve-1A) [reply tools/github-actions/audit/packaged-action/index.js:26190]: still, I don't think it is good that we have changes on something unrelated and that does not match the initial ts file

Comment (cpaulve-1A) [reply tools/github-actions/audit/packaged-action/index.js:26190]: Also, we should not lint generated files

File: nx.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (fpaul-1A) [reply nx.json:120]: I think we had issues in the past were glob with double quotes like `"**/*"` were evaluated before the command, ending up with issues (line too long or something, don't remember)

**Review by fpaul-1A — APPROVED** [review]

File: nx.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (kpanot) [reply nx.json:120]: I think indeed that Powershell evaluates `"**/*"` before executing the command which leads to issue, maybe the reason why `'` is not working need to be investigated :S.


> previously we were using [@nx/eslint:lint](https://nx.dev/nx-api/eslint/executors/lint) where we had to set `lintFilePatterns` I replaced it with that to have the same command on each package (we can still configure it for each if needed)
> or we can rediscussed it but it should not be part of this PR

I actually don't think it's a good solutions :S.
Having the patterns (and ignore patterns) in one (or several) dedicate files is much cleaner and configurable

[the remark this answers is not in the document]
Comment (matthieu-crouzet) [reply nx.json:120]: We can change it later :)
it's not part of this PR :)

**Pull request merged**